assets/js/blocks/mini-cart/style.scss

```diff
@@ -142,22 +142,31 @@ h2.wc-block-mini-cart__title {
 		display: flex;
 		gap: $gap;
 
-		.wc-block-mini-cart__footer-cart.wc-block-components-button {
+		.wc-block-components-button {
+			flex-grow: 1;
+			font-weight: 600;
+		}
+
+		.wc-block-components-button.outlined {
 			box-shadow: inset 0 0 0 1px currentColor;
 			color: currentColor;
 			display: none;
-			flex-grow: 1;
-			font-weight: 600;
+
+			&:hover,
+			&:focus {
+				color: $white;
+			}
+
+			&:focus {
+				box-shadow: 0 0 0 2px $studio-blue;
+				box-shadow: inset 0 0 0 1px $white, 0 0 0 2px $studio-blue;
+				outline: 3px solid transparent;
+			}
```

Comment:
I wonder if we could avoid repeating the styles from `assets/js/base/components/button/style.scss`, something like this: https://github.com/woocommerce/woocommerce-blocks/compare/fix/6575...fix/6575-style-suggestion?expand=1.

I know it doesn't simplify the code much because we need to use `:not()` in a couple of instances. But at the same time, it means the Mini Cart buttons are better aligned with the default styles, so I think it's probably a good thing in the long term.

Ideally, some day `assets/js/base/components/button/style.scss` will use `currentColor` instead of `$gray-900`, so we will be able to just reuse the styles from there, but that requires adding global styles to the Cart and Checkout blocks, so it's outside of our control right now.